packages/p2p/src/components/my-profile/my-profile-content.jsx

```diff
@@ -10,6 +10,7 @@ import PaymentMethods from './payment-methods';
 
 const MyProfileContent = () => {
     const { my_profile_store } = useStores();
+    const formik_ref = React.useRef();
```

Comment:
Using refs to obtain the Formik's state for Add PM form and Edit PM form outside the Formik component with Formik's `innerRef` prop

Reply:
This will also be used later with adding back button in create PM modal in another ticket